src/assets/stylesheets/application.scss

```
@@ -0,0 +1,20 @@
+// Variables
+@import 'variables';
+
+// Dependencies
+@import 'vendor/bootstrap/index';
+
+// Functions
+@import 'functions/sizing';
+
+// Mixins
+
+// Base
```

How about adding the base font?

```suggestion
// Base
@import 'base/fonts';
@import 'base/layouts';
```

```scss
// base/fonts.scss
@font-face {
  font-family: 'Neuzeit S LT Std';
  ...
}

// base/layouts.scss
body {
  font-family: 'Neuzeit S LT Std', serif;
}
```

---

Good idea :) 
Done in 5f5521d ✅